Example/package.json

```diff
@@ -0,0 +1,53 @@
+{
+  "name": "example",
+  "version": "0.0.1",
+  "private": true,
+  "scripts": {
+    "android": "react-native run-android",
+    "ios": "react-native run-ios",
+    "web": "webpack serve",
+    "macos": "react-native run-macos",
+    "lint": "eslint .",
+    "prettier": "prettier --write ./src/**/*.{tsx,ts}",
+    "start": "react-native start",
+    "test": "jest",
+    "tsc": "tsc"
+  },
+  "dependencies": {
+    "@babel/plugin-proposal-export-namespace-from": "^7.18.9",
+    "@svgr/webpack": "^8.1.0",
+    "file-loader": "^6.2.0",
+    "react": "18.2.0",
+    "react-native": "^0.73.6",
```

Comment:
Would be nice to bump the version to newest RN in some follow-ups. But then `macos` would probably stop working so idk.